packages/engine/src/assets/loaders/texture/TextureLoader.ts

```diff
@@ -94,7 +94,7 @@ class TextureLoader extends Loader<Texture> {
       if (signal?.aborted) return
       const isBitmap = i instanceof ImageBitmap
       const image = this.maxResolution && isBitmap ? getScaledBitmap(i, this.maxResolution) : i
-      if (!isBitmap) texture.flipY = this.flipped
+      texture.flipY = isBitmap ? !this.flipped : this.flipped
```

Comment:
I'm still working on this. It should be related to this code but skyboxes are still appearing upside down as well and I gotta find a solution for everyting.